integration/examples/helm-deployment/skaffold.yaml

```diff
@@ -3,7 +3,7 @@ kind: Config
 build:
   artifacts:
   - image: skaffold-helm
-deploy:
+manifests:
```

Comment:
I vaguely remember that @marlon-gamez told me that we want to keep helm in the old way and use helm as deployer? If we change this line, does this mean we use helm for render and use kubectl for deploy?

Reply:
Discussed w/ @MarlonGamez offline, having the schema use `manifests` is required to have the helm renderer work as expected (which is required for this test).

Reply:
ohhh I see. thanks.